costmodel/costmodel.go

```diff
@@ -174,6 +174,7 @@ const (
 					count_over_time(kube_pod_container_resource_requests{resource="nvidia_com_gpu", container!="",container!="POD", node!=""}[%s] %s) 
 					*  
 					avg_over_time(kube_pod_container_resource_requests{resource="nvidia_com_gpu", container!="",container!="POD", node!=""}[%s] %s)
+					* %f
```

Comment:
This will be subject to the same startup accuracy issues that we recently solved for CPU and RAM, but it's a fine solution for steady-state. Not saying we need to change it now, but putting it on everyone's radar.

Reply:
I know. we need to migrate to the gpu_allocation stat after this bakes for a while (that stat was recently added)